vispy/__init__.py

```diff
@@ -29,13 +29,20 @@
 
 from .util import config, set_log_level, keys, sys_info  # noqa
 from .util.wrappers import use  # noqa
```

Comment:
I wonder if this `test` function should be put in `util.wrappers` based on the docstring for that module (meant to avoid circular dependencies and other similar issues). Thoughts?

Reply:
You mean about moving whole test function, or only wraper?

Reply:
Moving the `test` wrapper you created here in `vispy/__init__.py` into `vispy/util/wrappers.py`.